### ISSUE TITLE
Anvil broken in 1.18

### ISSUE THREAD

**Issue by hamcoremusic** [Zrips/CMI#6418]
**Description of issue:**
Cannot use /anvil

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
[16:04:31] [Server thread/INFO]: HAMCORE issued server command: /anvil
[16:04:31] [Server thread/WARN]: java.lang.IllegalAccessException: Can not set final net.minecraft.world.inventory.ContainerPlayer field net.minecraft.world.entity.player.EntityHuman.bV to net.minecraft.world.inventory.ContainerAnvil
[16:04:31] [Server thread/WARN]:        at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:76)
[16:04:31] [Server thread/WARN]:        at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:80)
[16:04:31] [Server thread/WARN]:        at java.base/jdk.internal.reflect.UnsafeQualifiedObjectFieldAccessorImpl.set(UnsafeQualifiedObjectFieldAccessorImpl.java:79)
[16:04:31] [Server thread/WARN]:        at java.base/java.lang.reflect.Field.set(Field.java:799)
[16:04:31] [Server thread/WARN]:        at com.Zrips.CMI.Modules.Anvil.CMIAnvilGUI.open(CMIAnvilGUI.java:348)
[16:04:31] [Server thread/WARN]:        at com.Zrips.CMI.Modules.Anvil.CMIAnvilGUI.open(CMIAnvilGUI.java:301)
[16:04:31] [Server thread/WARN]:        at com.Zrips.CMI.commands.list.anvil.perform(anvil.java:80)
[16:04:31] [Server thread/WARN]:        at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:396)
[16:04:31] [Server thread/WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[16:04:31] [Server thread/WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149)
[16:04:31] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchCommand(CraftServer.java:800)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1937)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1776)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1757)
[16:04:31] [Server thread/WARN]:        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46)
[16:04:31] [Server thread/WARN]:        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:1)
[16:04:31] [Server thread/WARN]:        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.TickTask.run(SourceFile:18)
[16:04:31] [Server thread/WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:151)
[16:04:31] [Server thread/WARN]:        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.c(SourceFile:23)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1158)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.c(MinecraftServer.java:1)
[16:04:31] [Server thread/WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.y(SourceFile:125)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1137)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.y(MinecraftServer.java:1130)
[16:04:31] [Server thread/WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:134)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1114)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1038)
[16:04:31] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304)
[16:04:31] [Server thread/WARN]:        at java.base/java.lang.Thread.run(Thread.java:833
```

---

**Cmi Version (using`/cmi version`):** 
9.1.0.0
**Server Type (Spigot/Paperspigot/etc):** 
Spigot
**Server Version (using `/ver`):** 
Spigot 1.18-R0.1-SNAPSHOT

**Comment by mrfloris** [comment]
Do you also have this issue when you use paper build 25 ?

**Comment by hamcoremusic** [comment]
Seems to be a Spigot only issue right now. Will investigate more tonight.

**Comment by hamcoremusic** [comment]
Fixed on Paper-1.18-26
